bk-api/bk_api.py

```diff
@@ -585,17 +585,25 @@ def scp(source, node_id, target):
 
 
 
-def node_ssh(node_id, command, input_str=None):
+def node_ssh(node_id, command, input_str=None,quiet_mode=False):
+    proxy_cmd = f"ProxyCommand=ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@{BEEKEEPER_SSHD_HOST} -p 2201 -i /config/admin-key/admin.pem"
     ssh_cmd =  ["ssh",
-            #"-tt",
-            "-i", node_key , # this must be the key that allows ssh into the node , this key might be the same for all nodes
-            "-o", "UserKnownHostsFile=/dev/null",
-            "-o", "StrictHostKeyChecking=no",
-            "-o", "IdentitiesOnly=true",
-            "-o" ,f"ProxyCommand=ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@{BEEKEEPER_SSHD_HOST} -p 2201 -i /config/admin-key/admin.pem  netcat -U /home_dirs/node-{node_id}/rtun.sock",
-            f"root@foo:",
-            command]
-
+                #"-tt",
```

Comment:
1) this change seems unrelated to your VSN change and therefore be it's own commit in the event we need to revert either the VSN change or this ssh change

2) I would have done this a little different (for reference)

```
proxy_sub_cmd = f"ProxyCommand=ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@{BEEKEEPER_SSHD_HOST} -p 2201 -i /config/admin-key/admin.pem"
if quiet_mode:
    proxy_sub_cmd += " -q"
proxy_sub_cmd += f" netcat -U /home_dirs/node-{node_id}/rtun.sock"

proxy_cmd = ["-o", f"{proxy_sub_cmd}"]

ssh_options = proxy_cmd + [
    "-o",
    "UserKnownHostsFile=/dev/null",
    "-o",
    "StrictHostKeyChecking=no",
    "-o",
    "IdentitiesOnly=true",
]

ssh_cmd = [
    "ssh",
    "-i",
    node_key,  # this must be the key that allows ssh into the node , this key might be the same for all nodes
] + proxy_cmd
if quiet_mode:
    ssh_cmd.append("-q")
ssh_cmd.append(f"root@foo:")
ssh_cmd.append(command)
```

Reply:
1. The change is needed so that the cat command (https://github.com/iperezx/beekeeper/blob/master/bk-api/bk_api.py#L927) only contains the VSN value. If not you get a lot of output in stdout from the ssh connection. 
2. Isn't that just style? I wanted to keep it in one area instead of just scattered around the functon.

Reply:
1. i would still do the ssh change as a separate commit... or are you planning on merge squashing this?  i would prefer we did not do that.  i like to have separate well defined commits that you can reference back to and revert if needed.  this is why i do rebase on my PRs
2. see.. from my perspective the configuration of the proxy was already scattered around the function. ;)

Reply:
1. I have no preference. I can separate it.
2.😃